AES.Web/Models/ApplicationModels.cs

```diff
@@ -0,0 +1,321 @@
+        public bool TuesdayCheckBox { get; set; }
+
+        [Display(Name = "Tuesday From")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan TuesdayStartTime { get; set; }
+
+        [Display(Name = "Tuesday To")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan TuesdayEndTime { get; set; }
+
+        [Required]
+        [Display(Name = "Wednesday")]
+        public bool WednesdayCheckBox { get; set; }
+
+        [Display(Name = "Wednesday From")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan WednesdayStartTime { get; set; }
+
+        [Display(Name = "Wednesday To")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan WednesdayEndTime { get; set; }
+
+        [Required]
+        [Display(Name = "Thursday")]
+        public bool ThursdayCheckBox { get; set; }
+
+        [Display(Name = "Thursday From")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan ThursdayStartTime { get; set; }
+
+        [Display(Name = "Thursday To")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan ThursdayEndTime { get; set; }
+
+        [Required]
+        [Display(Name = "Friday")]
+        public bool FridayCheckBox { get; set; }
+
+        [Display(Name = "Friday From")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan FridayStartTime { get; set; }
+
+        [Display(Name = "Friday To")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan FridayEndTime { get; set; }
+
+        [Required]
+        [Display(Name = "Saturday")]
+        public bool SaturdayCheckBox { get; set; }
+
+        [Display(Name = "Saturday From")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan SaturdayStartTime { get; set; }
+
+        [Display(Name = "Saturday To")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan SaturdayEndTime { get; set; }
+
+        [Required]
+        [Display(Name = "Sunday")]
+        public bool SundayCheckBox { get; set; }
+
+        [Display(Name = "Sunday From")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan SundayStartTime { get; set; }
+
+        [Display(Name = "Sunday To")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:hh/mm/ss}")]
+        public TimeSpan SundayEndTime { get; set; }
+    }
+
+    public class WorkHistoryViewModel : ApplicationViewModel
+    {
+        [StringLength(25)]
```

Comment:
Double check that this, and all other ones like it match the database restrictions.